### PR TITLE
dynamically set tabindex -1 when details invisible. accessibility issue

### PR DESCRIPTION
**Pull request by williamfalconeruk**
## Purpose
To fix accessibility issue found by Axe

Fixes LPA-3669 and LPA-3693

## Approach
- fix javascript to dyamically add a `tabindex=-1` attribute to all child anchor and input tags inside of a detail, which can be toggled to show / hide. if hidden we add the tabindex as above, otherwise remove it.

## Checklist

* [X] I have performed a self-review of my own code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] The product team have tested these changes

### CODE REVIEW

**Review by Raffers — DISMISSED** [review]

File: service-front/assets/js/moj/moj.modules/moj.polyfill.js [hunk 1]
```diff
@@ -180,7 +218,9 @@
     }
 
     moj.Modules.DetailsPolyfill = {
+
         init: function () {
+            debugger;
```
Comment (johnnolan) [inline service-front/assets/js/moj/moj.modules/moj.polyfill.js:223]: cheeky

File: service-front/assets/js/moj/moj.modules/moj.polyfill.js [hunk 2]
```diff
@@ -13,6 +13,33 @@
 
     var NATIVE_DETAILS = typeof document.createElement('details').open === 'boolean'
 
+    /**
+     * restore tabbing on a list of focusable elements.
+     * fixes accessibility issue.
+     * @param elements
+     */
+    function restoreTabbing(elements){
```
Comment (johnnolan) [inline service-front/assets/js/moj/moj.modules/moj.polyfill.js:21]: If you wanted to make this smaller, you could change the name to toggle and pass in a bool that can set or remove inside.

File: service-front/assets/js/moj/moj.modules/moj.polyfill.js (hunk 1 above)
Comment (williamfalconeruk) [reply service-front/assets/js/moj/moj.modules/moj.polyfill.js:223]: 😁  oops - was supposed to remove that!

File: service-front/assets/js/moj/moj.modules/moj.polyfill.js (hunk 2 above)
Comment (williamfalconeruk) [reply service-front/assets/js/moj/moj.modules/moj.polyfill.js:21]: good point - was being explicit about it's purpose. will take a look.

**Review by Raffers — DISMISSED** [review]

**Review by Raffers — DISMISSED** [review]

**Review by Raffers — APPROVED** [review]

**Review by johnnolan — APPROVED** [review]

**Pull request merged**